styles/govuk/words-to-avoid.yml

```diff
@@ -0,0 +1,38 @@
+extends: substitution
+message: '%s'
+link: https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#words-to-avoid
+level: warning
+ignorecase: true
+swap:
+  agenda: Avoid using 'agenda', unless it's for a meeting 
+  advancing: Avoid using 'advancing'
+  collaborate: Use 'working with' instead of 'collaborate'
+  combating: Avoid using 'combating'
+  countering: Avoid using 'countering'
+  deliver: >
+    Avoid using 'deliver'. Pizzas, post and services are delievered; not abstract concepts like improvements or priorities
```

Comment:
Typo: 'delievered'

Concept: isn't it a bit hypocritical, if some of our people are... "Delivery Manager"s? 🙈

Reply:
I assumed our Delivery Managers fall under the "services are delivered" exception, but it does seem ambiguous.